operator/v1/zz_generated.crd-manifests/0000_50_csi-driver_01_clustercsidrivers-Default.crd.yaml

```diff
@@ -256,35 +256,6 @@ spec:
                   vSphere:
                     description: vSphere is used to configure the vsphere CSI driver.
                     properties:
-                      globalMaxSnapshotsPerBlockVolume:
```

Comment:
Well this doesn't look right 🤔 Have you scanned for the repository for any markers saying that this field relies on the gate we are removing?